src/components/Column.vue

```diff
@@ -76,6 +87,21 @@ export default defineComponent({
         task: {},
       });
     },
+    isSectionComplete(columnName: string) {
```

Comment:
The code that was already there is Vue2 style. Can you please update this component to be Vue3 (i.e. has a setup method)